diff --git a/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberSignatureTitle.js b/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberSignatureTitle.js
index 0a36b74..4a2b7fb 100644
--- a/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberSignatureTitle.js
+++ b/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberSignatureTitle.js
@@ -23,7 +23,9 @@ function MemberSignatureTitle({
   hideName,
   sig
 }) {
-  return /*#__PURE__*/React__default.default.createElement(React__default.default.Fragment, null, !hideName && sig.name !== '__type' ? sig.name : sig.kindString === 'Constructor signature' ? /*#__PURE__*/React__default.default.createElement(React__default.default.Fragment, null, sig.flags?.isAbstract && /*#__PURE__*/React__default.default.createElement("span", {
+  return /*#__PURE__*/React__default.default.createElement(React__default.default.Fragment, null, !hideName && sig.name !== '__type' ? 
+  React__default.default.createElement("span", null, (sig.modifiers ? `${sig.modifiers.join(' ')} ` : ''), React__default.default.createElement("b", null, sig.name))
+  : sig.kindString === 'Constructor signature' ? /*#__PURE__*/React__default.default.createElement(React__default.default.Fragment, null, sig.flags?.isAbstract && /*#__PURE__*/React__default.default.createElement("span", {
     className: "tsd-signature-symbol"
   }, "abstract "), /*#__PURE__*/React__default.default.createElement("span", {
     className: "tsd-signature-symbol"
@@ -39,8 +41,6 @@ function MemberSignatureTitle({
     className: "tsd-signature-symbol"
   }, "..."), param.name, /*#__PURE__*/React__default.default.createElement("span", {
     className: "tsd-signature-symbol"
-  }, (param.flags?.isOptional || 'defaultValue' in param) && '?', ': '), /*#__PURE__*/React__default.default.createElement(Type.Type, {
-    type: param.type
   })))), /*#__PURE__*/React__default.default.createElement("span", {
     className: "tsd-signature-symbol"
   }, ")"), sig.type && /*#__PURE__*/React__default.default.createElement(React__default.default.Fragment, null, /*#__PURE__*/React__default.default.createElement("span", {
diff --git a/node_modules/docusaurus-plugin-typedoc-api/lib/index.js b/node_modules/docusaurus-plugin-typedoc-api/lib/index.js
index 8c9d33a..0f9f3cc 100644
--- a/node_modules/docusaurus-plugin-typedoc-api/lib/index.js
+++ b/node_modules/docusaurus-plugin-typedoc-api/lib/index.js
@@ -55,7 +55,8 @@ const DEFAULT_OPTIONS = {
   routeBasePath: 'api',
   tsconfigName: 'tsconfig.json',
   typedocOptions: {},
-  versions: {}
+  versions: {},
+  pathToTypedocJSON: false,
 };
 
 async function importFile(file) {
@@ -152,7 +153,15 @@ function typedocApiPlugin(context, pluginOptions) {
 
           if (metadata.versionName === server.CURRENT_VERSION_NAME) {
             const outFile = path__default.default.join(context.generatedFilesDir, `api-typedoc-${pluginId}.json`);
-            await data.generateJson(projectRoot, entryPoints, outFile, options);
+            if (!options.pathToTypedocJSON) {
+              await data.generateJson(projectRoot, entryPoints, outFile, options)
+            }
+            else {
+              if (!fs.existsSync(context.generatedFilesDir)){
+                fs.mkdirSync(context.generatedFilesDir, { recursive: true });
+              }
+              fs.copyFileSync(options.pathToTypedocJSON, outFile);
+            } 
             packages = data.flattenAndGroupPackages(packageConfigs, await importFile(outFile), metadata.versionPath, options); // Versioned data is stored in the file system
           } else {
             const outDir = path__default.default.join(versionsDocsDir, `version-${metadata.versionName}`);
diff --git a/node_modules/docusaurus-plugin-typedoc-api/lib/plugin/sidebar.js b/node_modules/docusaurus-plugin-typedoc-api/lib/plugin/sidebar.js
index 8878837..e9d2822 100644
--- a/node_modules/docusaurus-plugin-typedoc-api/lib/plugin/sidebar.js
+++ b/node_modules/docusaurus-plugin-typedoc-api/lib/plugin/sidebar.js
@@ -12,7 +12,7 @@ const data = require('./data.js');
 
 function groupSidebarItems(map, groups) {
   const items = [];
-  const sortedGroups = groups.sort((a, b) => a.title.localeCompare(b.title));
+  const sortedGroups = groups;
 
   function getLastItemInGroup(index) {
     const length = sortedGroups[index]?.children?.length;
diff --git a/node_modules/docusaurus-plugin-typedoc-api/src/components/MemberSignatureTitle.tsx b/node_modules/docusaurus-plugin-typedoc-api/src/components/MemberSignatureTitle.tsx
index cfa5765..a60ca02 100644
--- a/node_modules/docusaurus-plugin-typedoc-api/src/components/MemberSignatureTitle.tsx
+++ b/node_modules/docusaurus-plugin-typedoc-api/src/components/MemberSignatureTitle.tsx
@@ -9,14 +9,14 @@ import { TypeParametersGeneric } from './TypeParametersGeneric';
 export interface MemberSignatureTitleProps {
 	useArrow?: boolean;
 	hideName?: boolean;
-	sig: JSONOutput.SignatureReflection;
+	sig: JSONOutput.SignatureReflection & { modifiers?: string[] };
 }
 
 export function MemberSignatureTitle({ useArrow, hideName, sig }: MemberSignatureTitleProps) {
 	return (
 		<>
 			{!hideName && sig.name !== '__type' ? (
-				sig.name
+				<span>{sig.modifiers ? sig.modifiers.join(' ') : ''}<b>sig.name</b></span>
 			) : sig.kindString === 'Constructor signature' ? (
 				<>
 					{sig.flags?.isAbstract && <span className="tsd-signature-symbol">abstract </span>}
@@ -35,13 +35,6 @@ export function MemberSignatureTitle({ useArrow, hideName, sig }: MemberSignatur
 					<span>
 						{param.flags?.isRest && <span className="tsd-signature-symbol">...</span>}
 						{param.name}
-
-						<span className="tsd-signature-symbol">
-							{(param.flags?.isOptional || 'defaultValue' in param) && '?'}
-							{': '}
-						</span>
-
-						<Type type={param.type} />
 					</span>
 				</React.Fragment>
 			))}
